infrastructure/terraform/demos/analytics/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "aws_region" {
+  description = "the AWS region in which resources are created, you must set the availability_zones variable as well if you define this value to something other than the default"
+  default     = "eu-central-1"
+}
+
+variable "aws_access_key" {
+  
+}
+
+variable "aws_secret_key" {
+  
+}
+
+variable "aws_zones" {
+  default = ["eu-central-1a", "eu-central-1b", "eu-central-1c"]
+}
+
+variable "vpc_name" {
+  default = "my-vpc"
+}
+
+variable "vpc_cidr" {
+  default = "10.0.0.0/16"
+}
+
+variable "private_subnets" {
+    default = "false"
+  
+}
+
+variable "tags" {
+  default = {}
+}
+
+variable "core_version" {
+  default = "0.31.0"
```

Comment:
I still need a way to make this point to `latest`

Reply:
Perhaps read the VERSION file (as the guide implies git clone) and render the content in a:
```
locals {
  version = data.template_file.version.rendered
}
```
Then you can use it as `local.version`.

An alternative I think would be to somehow use https://airy-core-binaries.s3.amazonaws.com/stable.txt